<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detection</class>
 <widget class="QWidget" name="Detection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>传感器信号</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="graph_L" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>811</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>信号选择</string>
   </property>
   <widget class="QCheckBox" name="en_L">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    image: url(:/icon/res/check_unchecked.png);
}

QCheckBox::indicator:checked {
    image: url(:/icon/res/check_checked.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>心跳通道</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>呼吸通道</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="en_L_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    image: url(:/icon/res/check_unchecked.png);
}

QCheckBox::indicator:checked {
    image: url(:/icon/res/check_checked.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>界面缩放</string>
   </property>
   <widget class="QRadioButton" name="zoom_h">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>横向伸缩</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="zoom_v">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>纵向伸缩</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="zoom_g">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>全局伸缩</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>X轴范围</string>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 滑动槽设置 */
QSlider::groove:horizontal {
	border:2px solid #ffffff;
	height:17px;
	background:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #009688, stop:1 #cddc39);
	margin:2px 0;
}

/* 滑块设置 */
QSlider::handle:horizontal {
	background:qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #8bc34a, stop:1 #cddc39);
	border:2px solid #ffffff;
	border-radius:5px;
	width:18px;
	
	/* 滑块上下边与滑道重合 */
	margin:-2px 0;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="L_hr">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(222, 222, 222);
    selection-background-color: darkgray;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>秒</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {    
    border-radius: 7.5px;
    background-color: #00bcd4;
	font: 75 12pt &quot;黑体&quot;;
	color: white;
}

QPushButton:hover{
	background-color: #03a9f4;
	color: white;
}

QPushButton:pressed{
	background-color: #2196f3;
    border-style: inset;
}2196f3</string>
   </property>
   <property name="text">
    <string>开始测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="com">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius:10px;background-color:rgb(250, 50, 50);font:75 12pt &quot;微软雅黑&quot;;color: white;}</string>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="graph_R" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>811</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>滤波</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>低通滤波：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="low_pass_filter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>13</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	font: 18pt &quot;微软雅黑&quot;;
    border: 2px solid gray;
    border-radius: 2px;
    min-width: 2em;
	border-image: url(:/icon/res/ico_btn_bg.png);
	color: rgb(255, 255, 255);
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;  
	border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 2px;
    border-bottom-right-radius: 2px;
	background-color: rgb(255, 255, 255);
}
QComboBox::down-arrow {
    image: url(:/icon/res/ui_down_arrow.png);
}
QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>400</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HZ以下</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filter_set">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {    
    border-radius: 5px;
    background-color: #00bcd4;
	font: 75 9pt &quot;黑体&quot;;
	color: white;
}

QPushButton:hover{
	background-color: #03a9f4;
	color: white;
}

QPushButton:pressed{
	background-color: #2196f3;
    border-style: inset;
}2196f3</string>
    </property>
    <property name="text">
     <string>开启滤波</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="tmp_version">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>560</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tmp_id">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>600</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">QCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
